bitrise.yml

```diff
@@ -100,7 +100,7 @@ workflows:
         inputs:
         - api_key: $DATADOG_API_KEY
         - app_key: $DATADOG_APP_KEY
-        - mobile_application_version_file_path: "/bitrise/src/example-app.apk"
```

Comment:
Having `mobile_application_version_file_path: "/bitrise/src/example-app.apk"` should have made the upload step fail.

But the bitrise CI wasn't red: this is a bug.

Can you temporarily revert 973d4cdeea382ac68c7750542c2f1a2fd5fcea60, and make sure it makes the CI fail?

To be robust, I think you should save the [exit code of this command](https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/blob/ee2af66557ab89c72e6bf8662d278722dfaf5e9d/upload-application.sh#L39-L45) with `$?`, and `exit` with that exit code.

Reply:
It succeeds 
https://app.bitrise.io/build/82622979-ff35-44ae-b01c-198fb354fb9a